packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -0,0 +1,165 @@
+# Configs and feature gates for solving the 1MB limit.
+
+## Introduction
+
+There is a current limitation regarding the size of the payload a Fluid client can send and receive. [The limit is 1MB per payload](https://github.com/microsoft/FluidFramework/issues/9023) and it is currently enforced explicitly with the `BatchTooLarge` error which closes the container.
+
+Version `client_v2.0.0-internal.2.3.0` introduces two features which can be used to work around this size limit, batch compression and compressed batch chunking. This document describes the prerequisites for safely leveraging the features, how to enable/disable them and a brief description of how they work.
```

Comment:
maybe remove reference to versions? Whatever version this readme shows up is a good one :)

Reply:
same below

Reply:
Should we keep it, though? I mean just to minimize confusion. Currently, [1.3.4](https://www.npmjs.com/package/fluid-framework/v/1.3.4) is considered latest https://www.npmjs.com/package/fluid-framework?activeTab=versions and what people would get with `npm i fluid-framework`.

Reply:
I think of versioning (and packaging) as completely separate from code. The extreme example - imaging that I fork FF and call it foo, and release foo@1.0. These numbers in code / readme do not make any sense. Same for adding readme.md with the new feature - we should not assume that it will show up in particular release that follows - it may get backed out and show up 2 releases later in code.
It all does not matter what nmp has - if you were to get to this version of the code, then it has nothing to do what's on Npm or anywhere else (latest version on a different feed could be different)